backend/api.test/Client/MissionTests.cs

```diff
@@ -39,11 +41,207 @@ public MissionTests(TestWebApplicationFactory<Program> factory)
+            {
+                Position = new Position
+                {
+                    X = 1,
+                    Y = 2,
+                    Z = 2
+                },
+                Orientation = new Orientation
+                {
+                    X = 0,
+                    Y = 0,
+                    Z = 0,
+                    W = 1
+                }
+            };
+
+            var installationQuery = new CreateInstallationQuery
+            {
+                InstallationCode = installationCode,
+                Name = installationCode
+            };
+
+            var plantQuery = new CreatePlantQuery
+            {
+                InstallationCode = installationCode,
+                PlantCode = plantCode,
+                Name = plantCode
+            };
+
+            var deckQuery = new CreateDeckQuery
+            {
+                InstallationCode = installationCode,
+                PlantCode = plantCode,
+                Name = deckName
+            };
+
+            var areaQuery = new CreateAreaQuery
+            {
+                InstallationCode = installationCode,
+                PlantCode = plantCode,
+                DeckName = deckName,
+                AreaName = areaName,
+                DefaultLocalizationPose = testPose
+            };
+
+            var installationContent = new StringContent(
+                JsonSerializer.Serialize(installationQuery),
+                null,
+                "application/json"
+            );
+
+            var plantContent = new StringContent(
+                JsonSerializer.Serialize(plantQuery),
+                null,
+                "application/json"
+            );
+
+            var deckContent = new StringContent(
+                JsonSerializer.Serialize(deckQuery),
+                null,
+                "application/json"
+            );
+
+            var areaContent = new StringContent(
+                JsonSerializer.Serialize(areaQuery),
+                null,
+                "application/json"
+            );
+
+            return (installationContent, plantContent, deckContent, areaContent);
+        }
+
+        private async Task<T> PostToDb<T>(string postUrl, StringContent content)
+        {
+            var response = await _client.PostAsync(postUrl, content);
+            Assert.True(response != null, $"Failed to post to {postUrl}. Null returned");
+            Assert.True(response.IsSuccessStatusCode, $"Failed to post to {postUrl}. Status code: {response.StatusCode}");
+            var responseObject = await response.Content.ReadFromJsonAsync<T>(_serializerOptions);
+            Assert.True(responseObject != null, $"No object returned from post to {postUrl}");
+            return responseObject;
+        }
+
+        private async Task<(string installationId, string plantId, string deckId, string areaId)> PostAssetInformationToDb(string installationCode, string plantCode, string deckName, string areaName)
```

Comment:
I may need some help since I've been working on this, but it seems the changes to the context do not carry over when using the endpoints later in the tests, likely due to the FlotillaDbContext being different from the one in Program.cs, even when using dependency injection to get it before using it to construct the DatabaseUtilities class. Not sure, but the context in installationService is at least null, while the one in robotService is not, for example